packages/wonder-blocks-testing/src/harness/types.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from "react";
+
+/**
+ * A adapter to be composed with our test harnass infrastructure.
+ */
+export type Adapter<TConfig> = (
```

Comment:
Probably should rename this to `HarnessAdapter` to avoid confusion with the fixture framework.